models/issue_comment.go

```diff
@@ -788,3 +789,39 @@ func DeleteComment(comment *Comment) error {
 	}
 	return nil
 }
+
+func fetchCodeComments(e Engine, issue *Issue, currentUser *User) (map[string]map[int64][]*Comment, error) {
+	pathToLineToComment := make(map[string]map[int64][]*Comment)
+	comments, err := findComments(e, FindCommentsOptions{
+		Type:    CommentTypeCode,
+		IssueID: issue.ID,
+	})
+	if err != nil {
+		return nil, err
+	}
+	if err = issue.loadRepo(e); err != nil {
+		return nil, err
+	}
+	for _, comment := range comments {
+		if err = comment.loadReview(e); err != nil && !IsErrReviewNotExist(err) {
```

Comment:
This should be remade to load reviews at once for all comments

Reply:
@lafriks Done